docs/whatsnew/1.4.0/1_4_0.rst

```diff
@@ -18,4 +18,34 @@ Check out the :ref:`hmm-section` tutorial and the
 
 .. figure:: hmm_hairpin.png
 
-   HMM analysis of a tethered DNA hairpin held at three different bead separations.
\ No newline at end of file
+   HMM analysis of a tethered DNA hairpin held at three different bead separations.
+
+Automatic bead cropping
+-----------------------
+
+Added :meth:`~lumicks.pylake.kymo.Kymo.estimate_bead_edges()` and :meth:`~lumicks.pylake.kymo.Kymo.crop_beads()` for roughly cropping the beads out of a kymograph. This can help when batch processing of kymographs.
```

Comment:
```suggestion
Added :meth:`~lumicks.pylake.kymo.Kymo.estimate_bead_edges()` and :meth:`~lumicks.pylake.kymo.Kymo.crop_beads()` for quickly crop the beads out of a kymograph using an estimate of the bead edges. This can help when batch processing of kymographs.
```